sql/dailyjobs.sql

```diff
@@ -0,0 +1,9 @@
+USE ouija;
+CREATE TABLE IF NOT EXISTS `dailyjobs` (
+    `date` datetime NOT NULL,
+    `platform` varchar(32) NOT NULL,
+    `branch` varchar(64) NOT NULL,
+    `numpushes` int NOT NULL,
+    `numjobs` int NOT NULL,
+    `sumduration` int NOT NULL
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
```

Comment:
I'm not certain how much data you're planning to retain in this table but you might want to consider creating indices on one or more of date, branch and platform to make the queries faster.